onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -836,14 +836,21 @@ def _where_input_wrangler(
         core_ops.aten_index_put_bool,
     ).skip(
         matcher=lambda sample: not (sample.args[0][0].dtype == torch.bool),
-        reason="this Aten overload only support tensor(bool) as args",
+        reason="this Aten overload only support tensor(bool) as indices",
     ),
     TorchLibOpInfo(
         "index_put",
         core_ops.aten_index_put,
     ).skip(
-        matcher=lambda sample: not (sample.args[0][0].dtype == torch.int64),
-        reason="this Aten overload only support tensor(int) as args",
+        matcher=lambda sample: not (
+            (sample.args[0][0].dtype == torch.int64)
+            # onnxruntime: MLFloat16 data type is not supported with ScatterND when reduction is 'add'
+            and (
+                sample.args[1].dtype != torch.float16
+                or not sample.kwargs.get("accumulate", False)
+            )
+        ),
+        reason="this Aten overload only support tensor(int) as indices and float32 when accumulate is True",
     ),
     TorchLibOpInfo("index_select", core_ops.aten_index_select),
```

Comment:
```suggestion
    TorchLibOpInfo("ops.aten.index_put", core_ops.aten_index_put),
    TorchLibOpInfo("index_select", core_ops.aten_index_select),
```

We then need to enable the test case like so